dissect/target/loaders/velociraptor.py

```diff
@@ -136,9 +136,14 @@ def map(self, target: Target) -> None:
 
 class VelociraptorDirectoryFilesystem(DirectoryFilesystem):
     def _resolve_path(self, path: str) -> Path:
-        return super()._resolve_path(quote(path, "$/").replace(".", "%2E"))
+        path = quote(path, "$/")
+        path = path.replace(".", "%2E", 1) if path.startswith(".") else path
```

Comment:
Does this work correctly in all cases? I'm not sure if `_resolve_path` gets called with an absolute path or relative path components. Does it still work as expected when there's a file or directory several levels deep in the filesystem tree?

Reply:
Hmm it didn't I fixed it by using `pathlib`, it gets called with a relative path.